docs/sources/administration/provisioning.md

```diff
@@ -273,6 +273,9 @@ notifiers:
     # or
     org_name: Main Org.
     is_default: true
+    send_reminders: true
+    frequency: 1m
```

Comment:
seems like a very low reminder frequency should this not be more like 1h ?

Reply:
Good point. Will push fix